### DISCUSSION TITLE
Is there a way to show both MONTH and YEAR  buttons in a month view?

### DISCUSSION THREAD

**Discussion by mikhail-yahorau** [wojtekmaj/react-calendar#900] [Q&A]
Is there a way to show both MONTH and YEAR  buttons in a month view?
So user can either click and select MONTH or click and select YEAR.

<img width="326" alt="image" src="https://github.com/wojtekmaj/react-calendar/assets/141716238/befdccbb-c795-46c4-8d3c-5018e4dbaead">

**Comment by rafelradan** [comment]
I have the same problem.
